docusaurus/docs/learn/core-concepts/security/enrollment.md

```diff
@@ -159,6 +159,140 @@ The default format is `[caName] - [commonName]`.
+
+- `enrollAttributeClaimsSelector` - a root level property name (e.g. `claims`) or a JSON pointer (e.g. `/claims`) that
+        points to a string or array of strings that will be used to populate the `attributes` field of the enrolling
+        identity. Defaults to no value set and identities will enroll without attributes.
+- `enrollAuthPolicyId` - the id of the authentication policy to use for the enrolling identity. Defaults to no value set
+        and the identity will enroll with the default authentication policy.
+- `enrollNameClaimsSelector` - the root level property name (e.g. `claims`) or a JSON pointer (e.g. `/claims`) that
+        points to a string that will be used to populate the `name` field of the enrolling identity.
+        Defaults to the subject property of the JWT (e.g. `/sub`)
+
+These values are included on the OpenZiti CLI command `ziti edge create ext-jwt-signer`.
+
+#### Identity Uniqueness
+
+The `claimsProperty` that selects the external id of the enrolling identity must be unique. Attempting to enroll
+with further duplicate external ids will result in an error. If desired, remove the previously enrolled identity
+to allow re-enrollment.
+
+#### Enrollment Process
+
+In order for Auto External JWT Signer Enrollment to happen, a client must be used that implements the enrollment flow.
+This process is similar to External JWT Signer authentication (e.g. OIDC or OAuth).
+
+1. The client queries the public External JWT Signers list on the Client API
+    ```http request
+    GET /edge/client/v1/external-jwt-signers
+    ```
+    
+    ```http response
+    200 OK
+    Content-Type: application/json
+    
+    {
+        meta: {...},
+        data: [
+            {
+                id: "20jgir0ji02",
+                  "name": "Example 1",
+                  "externalAuthUrl": "https://auth.example.com/login",
+                  "clientId": "native",
+                  "scopes": "openid profile email",
+                  "openIdConfigurationUrl": "https://auth.example.com/.well-known/openid-configuration",
+                  "audience": "myAudience",
+                  "targetToken": "ACCESS",
+                  "enrollToCertEnabled": true,
+                  "enrollToTokenEnabled": false,
+            },
+            ...
+        ]
+    }
+    ```
+2. The client presents a list to be selected from or selects a specific one.
+3. The client follows the authentication flow, enabling user input where necessary.
+4. The client receives a JWT from the external IdP.
+5. The client uses the JWT for enrollment. (see below)
+
+#### Enrollment Token Submission
+
+The client can submit the JWT to the controller for enrollment that results in either certificate based authentication
+(`enrollToCert` is true) or token based authentication (`enrollToToken` is true).
+
+
+##### Certificate Enrollment
+
+Certificate enrollment requries the creation of a CSR. The CSR can contain any values desired, however they will
```

Comment:
didn't catch it before but it'd be nice to link to wikipedia here to help people with "what is a CSR" also probably should spell out "certificate signing request (CSR)" at least one time